HTML/header_menu.html

```diff
@@ -4,13 +4,10 @@
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
wasn't this CSS unused ? or did I missed something ?

Reply:
I removed it.